Future.md

```diff
@@ -33,6 +33,29 @@ The following table shows mapping between current SARIF terminology as used for
 | Penetration Testing | Common | Common | Test | Test Step | Failure
 | Observability | [Common](https://opentelemetry.io/docs/concepts/context-propagation/#context) | Product Execution | [Signal](https://opentelemetry.io/docs/concepts/signals/) | [File / Line](https://opentelemetry.io/docs/specs/semconv/general/attributes/#source-code-attributes) | Alert
 
+## SARIF File Format
+
+Originally SARIF was embodied in **XML**. This format which although expressive, lacked ease of use.
```

Comment:
I suggest deleting the word "which" and adding a comma, as follows.
"This format, although expressive, lacked ease of use."